packages/react-baseline-inputs/src/Field.tsx

```diff
@@ -86,7 +87,11 @@ export function Field<T extends FieldProps>({
       {inline && render(inputProps)}
 
       {label && (
-        <label className={labelClassName} htmlFor={id}>
+        <label
+          className={labelClassName}
+          htmlFor={id}
+          id={error ? labelId : undefined}
```

Comment:
What was the rationale for conditionally giving this element an ID? Should we just always give it an ID?

Reply:
It's the way that the `label` gets associated with the `aria-labelledby` attribute.

It's not really needed in any other context so I made it conditional to cut down on extraneous markup

Reply:
That makes sense. I don't know why, but a conditional ID feels weird to me.